geomstats/euclidean_space.py

```diff
@@ -80,7 +77,8 @@ def log(self, point, base_point):
 
     def mean(self, points, weights=None):
         """
-        Weighted mean of the points.
+        The Frechet mean of (weighted) points is the weighted average of
```

Comment:
well - that's kind of obvious from the code. Seems like an overdescription

Reply:
Actually this comment is meant to highlight that the Frechet mean (a definition of mean on Riemannian manifolds) computed with the Euclidean metric boils down to our usual definition of mean as a weighted sum. This is a very desirable property but not necessarily obvious.